src/app/view/components/_common/iframe/iframe-component/iframe.component.ts

```diff
@@ -111,6 +117,67 @@ export class IframeComponent implements OnInit, OnDestroy {
         }
     }
 
+    private bindGlobalEvents(): void {
+        const events: string[] = [
+            'SAVE_FOLDER',
+            'UPDATE_FOLDER',
+            'DELETE_FOLDER',
+            'SAVE_PAGE_ASSET',
+            'UPDATE_PAGE_ASSET',
+            'ARCHIVE_PAGE_ASSET',
+            'UN_ARCHIVE_PAGE_ASSET',
+            'DELETE_PAGE_ASSET',
+            'PUBLISH_PAGE_ASSET',
+            'UN_PUBLISH_PAGE_ASSET',
+            'SAVE_FILE_ASSET',
+            'UPDATE_FILE_ASSET',
+            'ARCHIVE_FILE_ASSET',
+            'UN_ARCHIVE_FILE_ASSET',
+            'DELETE_FILE_ASSET',
+            'PUBLISH_FILE_ASSET',
+            'UN_PUBLISH_FILE_ASSET',
+            'SAVE_LINK',
+            'UPDATE_LINK',
+            'ARCHIVE_LINK',
+            'UN_ARCHIVE_LINK',
+            'MOVE_LINK',
+            'COPY_LINK',
+            'DELETE_LINK',
+            'PUBLISH_LINK',
+            'UN_PUBLISH_LINK',
+            'MOVE_FOLDER',
+            'COPY_FOLDER',
+            'MOVE_FILE_ASSET',
+            'COPY_FILE_ASSET',
+            'MOVE_PAGE_ASSET',
+            'COPY_PAGE_ASSET',
+            'DELETE_BUNDLE'
+        ];
+
+        const webSocketEvents$ = this.dotcmsEventsService
+            .subscribeToEvents<any>(events)
+            .pipe(takeUntil(this.destroy$));
+
+        webSocketEvents$
+            .pipe(filter(() => this.dotRouterService.currentPortlet.id === 'site-browser'))
+            .subscribe((event: DotEventTypeWrapper<any>) => {
+                this.loggerService.debug('Capturing Site Browser event', event.name, event.data);
+            });
+
+        webSocketEvents$
+            .pipe(
+                filter(
+                    (event: DotEventTypeWrapper<any>) =>
+                        event.name === 'DELETE_BUNDLE' &&
+                        this.dotRouterService.currentPortlet.id === 'publishing-queue' &&
```

Comment:
`this.dotRouterService.currentPortlet.id === 'publishing-queue'` this is not needed. since the refresh will be handled by the jsp

Reply:
True this! great catch.